src/writer.jl

```diff
@@ -161,11 +161,24 @@ function write(path::AbstractString, obj; force=false)
 
     # Close .shp file
     close(io)
-    
+
     # Write .shx file
     index_handle = IndexHandle(header, shx_indices)
     Base.write(paths.shx, index_handle)
 
+    # Write .prj file
+    crs = try
+        GI.crs(obj)
+    catch
+        nothing
+    end
+    if !isnothing(crs)
+        try
+            crs isa GeoFormatTypes.ESRIWellKnownText && Base.write(paths.prj, crs.val)
```

Comment:
Here we can actually run `convert(ESRIWellKnownText, crs)` if it isnt `ESRIWellKnownText`.

It needs ArchGDAL to be loaded to actually do the conversion (withsome sanctioned type-pyracy) - but  we should make GeoFormatTypes.jl specifically tell you to run `using ArchGDAL` in the error.

Reply:
I disagree here, we shouldn't depend on ArchGDAL like that.

Reply:
Sorry I don't get it... there is no dependency.

This just means it won't error if ArchGDAL is loaded, but will error if it isn't.

Reply:
I agree with @rafaqz.  The sanctioned ArchGDAL pirated `convert` methods are hard to discover.  I changed the code to run the `convert` but provide a useful warning (not error) when the conversion fails.